OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -97,6 +97,9 @@ public class AircraftInfo : ITraitInfo, IPositionableInfo, IFacingInfo, IMoveInf
 		[Desc("Does this actor cancel its previous activity after resupplying?")]
 		public readonly bool AbortOnResupply = true;
 
+		[Desc("Can this actor be given an explicit land order using the force-move modifier?")]
+		public readonly bool ForceLanding = true;
```

Comment:
Would this make sense as a `FlightDynamics` flag?

Reply:
That was my first instinct, until halfway through writing it I realized that if you have an exception unit you can't just inherit the `FlightDynamics` from the default and change just the one parameter. You have to repeat all the completely unrelated parts of `FlightDynamics` as well. 
Furthermore, you can't have a parameter default to true, so unless I reword this into something awkward every unit in every mod will have to be adapted to explicitly allow it. Combined  with the first point, there is lot's of potential for bugs due to oversights with units that already have some exceptions. 

To be honest, I'm not sure if `FlightDynamics` was actually a good idea in the first place.

Reply:
Wouldn't something like `DisableForceLanding` work for inverting the default?

Reply:
This raises a good point that, now thinking about it, I am also not completely happy with.

Merging the various flags that define the actual movement characteristics was IMO a good thing. The potential problem is that we included flags that aren't really related to this -  `MoveIntoShroud`, maybe also `TakeOffOnResupply` and `TakeOffOnCreation`.

So I think it does make sense to keep `ForceLanding` separate, and we should also move `MoveIntoShroud` and maybe `TakeOffOn*` back out too. What do you think @reaperrr?

Reply:
I'm fine with moving `MoveIntoShroud` back. I'm a bit divided on moving back the `TakeOff*`s, because strictly speaking then it would be more consistent to move the `TurnTo*`s back as well, and possibly even `VTOL` as that's als take-off/land-related. At which point on of the main purposes of that PR - reducing the amount of fields - would be more than half-way defeated...

Could we maybe compromise like this? Leave the _order_-related fields as booleans (move back `MoveIntoShroud` and let this new `DisableForceLanding` stay) and maybe split the take-off/landing-related dynamics to a separate `TakeOffAndLandDynamics`?
That would at least reduce the number of flags you have to re-list every time you change a flag and make the respective `*Dynamics` field name more accurate concerning what it does.

Reply:
My thought wrt splitting the two `TakeOffOn*` flags is that they relate to *when* the aircraft should take off rather than *how* it takes off. IMO the `VTOL` and `TurnToLand`/`TurnToDock` flags are just as much a part of the dynamics as `Slide` or `Hover`, so I don't like the idea of moving them into their own field.

We could have a `TakeOffTriggers` instead, but if this is going to be controversial then it may be easier to keep them on `FlightDynamics`...

Reply:
Changed the name to `EnableForceLanding`.

Revising the other flags and booleans is out of scope for this PR. However, to add my perspective:

As far as I'm concerned only `VTOL`, `Slide` and `Hover` (and maybe `TurnToLand`) belong in `FlightDynamics`. 

`TurnToDock` specifically relates to whether there is a preferred orientation for docking at an airfield or helipad and might be better of as a characteristic of those actors. For example, what might be an appropriate orientation for landing at an airfield might not be for landing at a service depot. Such a parameter would be relevant for ground units as well.

`TurnToLand` is currently not used by any unit at all and seems to exist only for legacy behaviour.

I can get behind a `TakeOffOn` enum if we indeed plan to make more variants. I'm just wondering if it would be feasible to make a `TakeOffOnCondition` parameter instead and to do everything with conditions to make it completely generic.

Of the three flags left, two might in the future be replaced by integer parameters (`IdleSpeed` and `SlideSpeed`). So that leaves us with just `VTOL`...

Reply:
> TurnToLand

Even if we keep the logic, we can replace the boolean with a `LandFacing` int, as using `InitialFacing` is a bit counter-intuitive anyway.

Reply:
`InitialFacing` really should be removed anyway - it should be up to the thing creating the actor to define its initial facing.